users/peterfalken/peterfalken.c

```diff
@@ -0,0 +1,11 @@
+// Copyright 2023 Peter.Falken (@PeterFalken)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "peterfalken.h"
+
+// Switch layer
+#ifndef CUSTOM_LAYER_STATE
+layer_state_t layer_state_set_user(layer_state_t state) {
```

Comment:
The core [tri layer](https://docs.qmk.fm/#/feature_tri_layer) feature should be used here instead. And this file can be removed.

Reply:
Thanks for the review @waffle87. I left it like this to accommodate the Adafruit - MacroPad & the Work Louder - Micro, which at the moment don't use the tri_layer approach. To be honest, my System76 - Launch_1 was just recently updated to use the _LOWER, _UPPER & _ADJUST layers. Will probably rewrite this function at a later time.